wordpress-mobile/WordPress-iOS
# Issue/set media type on creation

Make sure Media objects have the correct type when they are created.

To test:
 - Start the app
 - Open the media library of a site
 - Add new media
 - Check that everything works ok.
